package.json

```diff
@@ -12,7 +12,7 @@
     "copier:root": "copier -a .copier-project.yaml -s apps/api update",
     "bootstrap": "dotenv -c local -- run-s bootstrap:dev bootstrap:users",
     "bootstrap:dev": "wsrun -msc bootstrap resources resources.dev",
-    "bootstrap:prod": "wsrun -msc bootstrap resources",
+    "bootstrap:prod": "wsrun -msc bootstrap resources resources.prod",
```

Comment:
Should this be changed to:

```
"bootstrap:live": "wsrun -msc bootstrap resources resources.live",
```

Or be removed if not used?

Reply:
Of course!